docs/markdown/Python-module.md

```diff
@@ -238,6 +238,63 @@ module.
 
 It exposes the same methods as its parent class.
 
+## A note about Debian installation paths
+
+Debian and derivatives (Ubuntu, Linux Mint etc.) [change default installation
+and import paths][debian-dist-packages] in system interpreter. The problem is,
```

Comment:
This needs an article. "the system interpreter"?  But I'm not sure what this means?  "the system Python interpreter"?

Reply:
Fixed.